docs/source/ops/system_architecture_faq.rst

```diff
@@ -0,0 +1,89 @@
+
+**Further Reading:**
+
+- `Encryption Keys <../canton/usermanual/security.html#participant-encryption-key>`__
+- `View Encryption Keys <../canton/usermanual/security.html#view-encryption-key>`__
+- `Cryptographic Key Usage <../canton/usermanual/security.html#cryptographic-key-usage>`__
+
+.. _where_holden_source:
+
+Where does "the golden source" of Daml Ledger data live in Canton?
+//////////////////////////////////////////////////////////////////
+
+The short answer is that Daml Ledger data lives both on the Canton participant nodes, and on the squencer backend, meaning the blockchain or database enabled by the driver. The data is stored in the two places in different ways, but fully consistent thanks ot Canton's deterministic execution model.
+
+All communication between Canton nodes, including the confirmation requests for transactions and the resulting confirmations and rejections, are stored on sequencer backend. Since Daml and Canton are built around deterministic execution, you can thus consider that data on the sequencer backend, together with the Participant Encryption Keys, to be a complete copy of the Daml Ledger. 
+
+On the flip side, each Participant node stores its view of the Daml Ledger in an unencrypted, indexed, and queriable format. The set of all participant nodes jointly holds the entire ledger state and history.
+
+**Further Reading:**
+
+- `Transaction processing in Canton <../canton/architecture/overview.html#transaction-processing-in-canton>`__
+
+
+How is Canton able to recover from data loss?
+/////////////////////////////////////////////
+
+As discussed in ":ref:`where_holden_source`", the ledger data lives in two places, once encrypted in the sequencer backend, and once unencrypted spread between participant nodes. As long as you have a complete copy in either place, you can recover and continue operation.
+
+As long as the entire ledger history in the sequencer backend is available, and you hold a participant's Participant Encryption Keys, it is possible to recover the participant from the underlying sequencer backend. So if you use a blockchain as the sequencer backend, and can ensure that that blockchain stays available and uncorrupted, you can always recover from participant data loss.
+
+Should your sequencer backend go down, but all participants are still up and running, you can continue running the system by coordinating all participant nodes to “transfer” active contracts to a new domain, with a new sequencer backend.
+
+Should the sequencer backend no longer have the full ledger history, for example due to a domain switch, or because of deliberate Ledger Pruning, participants can still recover from a combination of the partial sequencer backend and a state snapshot. Such a snapshot can come either from a backup, or from its peers.
```

Comment:
```suggestion
Should the sequencer backend no longer have the full ledger history, for example due to a domain switch, or because of deliberate ledger pruning, participants can still recover from a combination of the partial sequencer backend and a state snapshot. Such a snapshot can come either from a backup, or from the participants' peers.
```